.github/workflows/ci.yml

```
@@ -53,7 +53,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Check for Docker images
         id: check_images
-        run: echo "::set-output name=images::$(docker images -q | wc -l)"
+        run: echo "{name}={images::$(docker images -q | wc -l)}" >> $GITHUB_OUTPUT
```

The extra curly braces don't look right here. I'd assume that this could simply be:
```suggestion
        run: echo "images=$(docker images -q | wc -l)" >> $GITHUB_OUTPUT
```

---

Side note: This step in GitHub Actions CI is not very useful in the first place since I assume that there would always be some pre-existing images. In any case, the initial state in GitHub Actions CI is the same each time, so it doesn't make much sense to have condition logic like this.

---

My bad, I took it from the templating I saw in the docs, thinking it was the actual pattern required. I'll remove them.